images/paranoia/tests/01-version.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+
+if [[ "${IMAGE_NAME}" == "" ]]; then
+    echo "Must set IMAGE_NAME environment variable. Exiting."
+    exit 1
+fi
+
+docker run --rm "${IMAGE_NAME}" --help 2>&1 | grep "Paranoia is a command-line tool"
```

Comment:
Can we test something more thorough than just the version?

Reply:
I just added a test for the "export" capability of paranoia against the `alpine` image.